config/overlays/test-infra/components/auth/auth-tokens-secret.yaml

```diff
@@ -5,5 +5,5 @@ metadata:
 type: Opaque
 stringData:
   tokens.csv: |
-    test-admin-token,admin,1001,"system:masters"
+    test-admin-token,admin,admin,"system:masters"
     test-user-token,test-user,1002,"system:authenticated"
```

Comment:
I'd update this value as well since it would run into the same issue you had when using the admin token.